src/SamplesApp/SamplesApp.Droid/Properties/AssemblyInfo.cs

```diff
@@ -43,3 +43,6 @@
 [assembly: UsesPermission("android.permission.WRITE_EXTERNAL_STORAGE")]
 [assembly: UsesPermission("android.permission.CAMERA")]
 [assembly: UsesPermission("android.permission.RECORD_AUDIO")]
+
+[assembly: UsesFeature("android.software.leanback", Required = false)]
+[assembly: UsesFeature("android.hardware.touchscreen", Required = false)]
```

Comment:
Same question, if this is necessary, should it be documented and/or included in templates?

Reply:
Note: This is included for Xamarin but not net6.0+?